src/pods/canvas/components/table/database-table.business.ts

```diff
@@ -0,0 +1,7 @@
+export const calculateColumNameWidth = (
```

Comment:
Create an issue indicating to implement unit tests on this method, it can be a good starting point for new mats that join into the project